voluncheer/voluncheer/settings.py

```diff
@@ -91,6 +94,40 @@
 ]
 
 WSGI_APPLICATION = "voluncheer.wsgi.application"
+ASGI_APPLICATION = "voluncheer.asgi.application"
+
+
+REDIS_PROD_MAIN_PORT = os.getenv("REDIS_PROD_MAIN_PORT")
```

Comment:
```suggestion
REDIS_MAIN_PORT = os.getenv("REDIC_MAIN_PORT")
```

Reply:
Then delete the prod/dev environment branches below in favor of a single branch when on aws. The don't have any difference other that the address/port.

Reply:
Actually, they run on two different redis servers on the backend. I think there are good reasons to set up 2 servers so downing the develop doesn't result in the downing of production. Is it too expensive?

Reply:
But they are unique to the environment they connect to, right? The prod one connects to main and the dev one connects to develop I assume.